index.rst

```diff
@@ -703,70 +684,69 @@ Changing the configuration requires changing the Helm chart or the generated ``C
-                   "<image tag>": "<human readable name>"
-               },
+               "reference": "<full Docker reference>",
+               "tag": "<tag name>"
+               "aliases": ["<tag name>", "<tag name>"],
                "name": "<human readable name>",
 	       "digest": "<image digest>",
                "prepulled": true
            },
            "latest-weekly": {
-               "path": "<full Docker image path>",
-               "tags": {
-                   "<image tag>": "<human readable name>"
-               },
+               "reference": "<full Docker reference>",
+               "tag": "<tag name>"
+               "aliases": ["<tag name>", "<tag name>"],
                "name": "<human readable name>",
 	       "digest": "<image digest>",
                "prepulled": true
            },
            "latest-daily": {
-               "path": "<full Docker image path>",
-               "tags": {
-                   "<image tag>": "<human readable name>"
-               },
+               "reference": "<full Docker reference>",
+               "tag": "<tag name>"
+               "aliases": ["<tag name>", "<tag name>"],
                "name": "<human readable name>",
 	       "digest": "<image digest>",
                "prepulled": true
            },
            "latest-release": {
-               "path": "<full Docker image path>",
-               "tags": {
-                   "<image tag>": "<human readable name>"
-               },
+               "reference": "<full Docker reference>",
+               "tag": "<tag name>"
+               "aliases": ["<tag name>", "<tag name>"],
                "name": "<human readable name>",
 	       "digest": "<image digest>",
                "prepulled": true
            },
            "all": [
                {
-                   "path": "<full Docker image path>",
-                   "tags": {
-                       "<image tag>": "<human readable name>"
-                   },
+                   "reference": "<full Docker reference>",
+                   "tag": "<tag name>"
+                   "aliases": ["<tag name>", "<tag name>"],
                    "digest": "<image digest>",
                    "name": "<human readable name>",
                    "prepulled": false
                }
            ]
        }
 
+    Each of the top-level object keys is the string literal exactly as shown.
+    The alias information may not be complete.
+    (In other words, the image may have additional aliases that are not shown.)
+
 ``GET /nublado/spawner/v1/prepulls``
     Returns status of the known prepull configurations.
     The response is a JSON object with three keys.
 
     The first key is ``config``, which contains the prepull configuration.
-    This is identical to the configuration blocks given above, but converted to JSON.
+    This is identical to the configuration blocks given above, but converted to JSON and using snake-case instead of camel-case for keys.
```

Comment:
I assume you're doing that rather than "snake_case instead of CamelCase" just to make people like me flinch.